src/main/java/fi/otavanopisto/kuntaapi/server/integrations/ptv/AbstractPtvProvider.java

```diff
@@ -52,5 +84,565 @@
+      if (serviceLocationChannel != null) {
+        result.add(serviceLocationChannel);
+      }
+    }
+
+    return result;
+  }
+  
+  protected List<WebPageChannel> translateWebPageChannels(List<fi.otavanopisto.restfulptv.client.model.WebPageChannel> ptvWebPageChannels) {
+    if (ptvWebPageChannels == null) {
+      return Collections.emptyList();
+    }
+
+    List<WebPageChannel> result = new ArrayList<>();
+    for (fi.otavanopisto.restfulptv.client.model.WebPageChannel ptvWebPageChannel : ptvWebPageChannels) {
+      WebPageChannel webPageChannel = translateWebPageChannel(ptvWebPageChannel);
+      if (webPageChannel != null) {
+        result.add(webPageChannel);
+      }
+    }
+
+    return result;
+  }
+
+  protected ElectronicChannel translateElectronicChannel(fi.otavanopisto.restfulptv.client.model.ElectronicChannel ptvElectronicChannel) {
+    if (ptvElectronicChannel == null) {
+      return null;
+    }
+    
+    OrganizationId organizationKuntaApiId = translateOrganizationId(ptvElectronicChannel.getOrganizationId());
+    if (organizationKuntaApiId == null) {
+      return null;
+    }
+    
+    ElectronicServiceChannelId channelPtvId = new ElectronicServiceChannelId(PtvConsts.IDENTIFIFER_NAME, ptvElectronicChannel.getId());
+    ElectronicServiceChannelId channelKuntaApiId = idController.translateElectronicServiceChannelId(channelPtvId, KuntaApiConsts.IDENTIFIER_NAME);
+    if (channelKuntaApiId == null) {
+      logger.severe(String.format("Could not translate electronic channel id %s into Kunta API id", channelPtvId.getId()));
+      return null;
+    }
+    
+    ElectronicChannel result = new ElectronicChannel(); 
+    result.setAttachments(translateAttachments(ptvElectronicChannel.getAttachments()));
+    result.setDescriptions(translateLocalizedItems(ptvElectronicChannel.getDescriptions()));
+    result.setId(channelKuntaApiId.getId());
+    result.setLanguages(ptvElectronicChannel.getLanguages());
+    result.setNames(translateLocalizedItems(ptvElectronicChannel.getNames()));
+    result.setOrganizationId(organizationKuntaApiId.getId());
+    result.setPublishingStatus(ptvElectronicChannel.getPublishingStatus());
+    result.setRequiresAuthentication(ptvElectronicChannel.getRequiresAuthentication());
+    result.setRequiresSignature(ptvElectronicChannel.getRequiresSignature());
+    result.setServiceHours(translateServiceHours(ptvElectronicChannel.getServiceHours()));
+    result.setSignatureQuantity(ptvElectronicChannel.getSignatureQuantity());
+    result.setSupportContacts(translateSupportContacts(ptvElectronicChannel.getSupportContacts()));
+    result.setType(ptvElectronicChannel.getType());
+    result.setUrls(translateLanguageItems(ptvElectronicChannel.getUrls()));
+    result.setWebPages(translateWebPages(ptvElectronicChannel.getWebPages()));
+    
+    return result;
+  }
+
+  private OrganizationId translateOrganizationId(String ptvOrganizationId) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused private "translateOrganizationId" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUnusedPrivateMethod)